test/cypress/stubs/webhooks-stubs.js

```diff
@@ -21,7 +21,7 @@ function getWebhookMessagesListSuccess (opts = {}) {
   return stubBuilder('GET', path, 200, {
     query: {
       page: opts.page || 1,
-      ...opts.status && { status: opts.status }
+      ...opts.status && { status: opts.status.toUpperCase() }
```

Comment:
I'll admit I have little context on this but would it not make more sense to update the status where this is called, rather than obscure that it becomes upper case?

Reply:
Fair point, I've updated at the source.